### ISSUE TITLE
update OII branch with latest changes

### ISSUE THREAD

**Issue by joehobson** [navnorth/wp-oet-theme#5]
Update the OII branch from master so we have the latest keyboard navigation changes in there. You should have push/pull access now so you can push directly to the branch rather than going through a pull request.

**Comment by johnpaulbalagolan** [comment]
closes #5
